mastodon_archive/__init__.py

```diff
@@ -88,6 +88,9 @@ def main():
         help='download media referred to by toots in your archive')
     parser_content.add_argument("user",
                                 help='your account, e.g. kensanata@octogon.social')
+    parser_content.add_argument("--combine",
+                                action="store_true",
+                                help="combine JSON files, in case of split archives")
```

Comment:
Is this help message clear enough?

Reply:
Good enough for me! The splitting help message never says "JSON files" so "combine archives in case they are split" would have also worked.

Reply:
Will do.

Reply:
Done in d2f2863